tests/src/psa_exercise_key.c

```diff
@@ -692,7 +692,7 @@ int mbedtls_test_psa_exported_key_sanity_check(
         TEST_EQUAL(mbedtls_asn1_get_tag(&p, end, &len,
                                         MBEDTLS_ASN1_SEQUENCE |
                                         MBEDTLS_ASN1_CONSTRUCTED), 0);
-        TEST_EQUAL(len, end - p);
+        TEST_EQUAL(len, (uintptr_t) end - (uintptr_t) p);
```

Comment:
I'm not sure this is the right way to do it. AIUI, `end - p` has type `ptrdiff_t`, i.e., signed, so I think you should be able to write as `(size_t)(end - p)`

Reply:
After this I had put casts in the TEST_EQUAL macro so in fact I can remove the casts on this line entirely

Reply:
Great, that's the only bit I would have wanted a second reviewer for so I'm now marking single-reviewer.